Editor/ObjectMapping/AnimationObjectMapper.cs

```diff
@@ -212,5 +213,18 @@ public MappedGameObjectInfo(ObjectMapping objectMapping, string? newPath, Before
                 return new[] { (gameObjectInfo.NewPath, type, propertyName) };
             }
         }
+
+        public bool TryGetMappedVrmFirstPersonFlag(Renderer component, out VrmFirstPersonFlag vrmFirstPersonFlag)
+        {
+            var componentInfo = _objectMapping.GetComponentMapping(component.GetInstanceID());
```

Comment:
mapあとのpathで`Transform.Find().renderer`を使う必要があるようなので、であればstringでpathを取得して、そこからGameObjectInfoを取得し、こんな感じに`GetComponentByType(typeof(Renderer))`するようにしてください。

```suggestion
        public bool TryGetMappedVrmFirstPersonFlag(string path, out VrmFirstPersonFlag vrmFirstPersonFlag)
        {
            vrmFirstPersonFlag = default;
            var gameObjectInfo = GetGameObjectInfo(path);
            if (gameObjectInfo == null) return false;
            var (_, componentInfo) = gameObjectInfo.GetComponentByType(typeof(Renderer));
```

Reply:
:pray: https://github.com/anatawa12/AvatarOptimizer/pull/1376/files/1015e146c981251ae9a19e815f57660c55eda20f..0e6777e75a0ffcccec8bdc7e07bfbd6e5e553732